openedx_webhooks/github_views.py

```diff
@@ -71,13 +71,6 @@ def hook_receiver():
     if who == get_bot_username() and "comment" in event:
         return "No thanks", 202
 
-    # This can't authenticate with Jira now, so don't do it:
```

Comment:
Although we can't use this code anymore, let's leave the comment as a reminder of what work got dispatched there.  In reading the code recently, it was helpful to see what that dispatch used to do.  I promise I'll be redoing this stuff soon. :)